h2o-algos/build.gradle

```diff
@@ -11,6 +11,15 @@ dependencies {
 
 apply from: "${rootDir}/gradle/dataCheck.gradle"
 
+task testSSLEncryption(type: Exec) {
+  dependsOn cpLibs, jar, testJar
+  if(project.hasProperty('test_ssl')) {
```

Comment:
I would also prefer to follow the convention: `doTestSsl` as we have for the rest of stuff. See `gradle.properties` file.